FWCore/Services/plugins/CondorStatusUpdater.cc

```diff
@@ -419,6 +425,7 @@ void CondorStatusService::fillDescriptions(ConfigurationDescriptions &descriptio
       ->setComment("Interval, in seconds, to calculate event rate over (using EMA)");
   desc.addOptionalUntracked<std::string>("tag")->setComment(
       "Identifier tag for this process (a value of 'Foo' results in ClassAd attributes of the form 'ChirpCMSSWFoo*')");
+  desc.addOptionalUntracked<bool>("disable", false)->setComment("Disable this service");
```

Comment:
How about inverting the logic along
```suggestion
  desc.addOptionalUntracked<bool>("enable", true)->setComment("Enable this service");
```
? (just thinking if avoiding the double negation of "not disabling" would be more clear)